VisualPinball.Engine/VisualPinball.Engine.csproj

```diff
@@ -88,9 +88,18 @@
 		<Reference Include="System.ServiceModel" />
 		<Reference Include="System.Transactions" />
 		<Reference Include="System.Xml" />
+		<Reference Include="Unity.Entities.Hybrid, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">

```

Comment:
Are these assembly reference needed ? some are pointing to vpx_import_test

Reply:
You're right this has nothing to do here.